src/views/Tags.vue

```diff
@@ -27,37 +27,38 @@
 	</EmptyContent>
 
 	<!-- Folder content -->
-	<Grid v-else-if="!loading">
+	<div v-else-if="!loading">
 		<Navigation
 			key="navigation"
 			:basename="path"
 			:filename="'/' + path"
 			:root-title="rootTitle" />
-
-		<!-- Tags list -->
-		<template v-if="isRoot">
-			<Tag v-for="id in tagsNames"
-				:key="id"
-				v-bind="tags[id]"
-				:fileid="id"
-				:basename="tags[id].displayName" />
-		</template>
-
-		<!-- Content list -->
-		<template v-else>
-			<EmptyContent v-if="isEmpty" key="emptycontent" illustration-name="empty">
-				{{ t('photos', 'No tags yet') }}
-				<template #desc>
-					{{ t('photos', 'Photos with tags will show up here') }}
-				</template>
-			</EmptyContent>
-
-			<File v-for="file in fileList"
-				:key="file.fileid"
-				:list="fileList"
-				v-bind="file" />
-		</template>
-	</Grid>
+		<Grid>
```

Comment:
I don't see how this is relevant to this pull request?

Reply:
I updated the css of the navigation to fit the new grid, and it is impacting the other pages, so I removed the navigation from being inside the grid :)

Reply:
It is due to the new styles applied to titles to fit the new grid. I had to fix them on other pages.